browser/policy/DEPS

```diff
@@ -1,5 +0,0 @@
-specific_include_rules = {
-  "brave_talk_policy_unittest.cc": [
-    "+components/sync_preferences",
```

Comment:
why is that removed?
It still has 
`#include "components/sync_preferences/testing_pref_service_syncable.h"`